src/redundantaudioencoder/RedundantAudioEncoder.ts

```diff
@@ -1480,3 +1480,15 @@ namespace RedundantAudioEncoder {
     windowSize: number;
   }
 }
+
+/* istanbul ignore next */
+/**
+ * This function is only meant to be called within the RED worker to initialize and start RED functionality.
+ */
+function startRedWorker(): void {
+  RedundantAudioEncoder.shouldLogDebug = true;
```

Comment:
This should be false, yes?

Reply:
We want the RED worker to be able to emit logs, so this should be true

Reply:
The encoder file only has low frequency logs at the moment, so this shouldn't overload the main thread with too many messages

Reply:
Can we just based on the Log level defined ?

Reply:
Updated to configure this based on log level